### PR TITLE
[tmux/lt]

### PR DESCRIPTION
**Pull request by zygimantus**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: lt-lt/tmux-lt.html.markdown [hunk 1]
```diff
@@ -0,0 +1,252 @@
+---
+category: tool
+tool: tmux
+filename: tmux.json
+lang: lt-lt
+contributors:
+    - ["Zygimantus", "https://github.com/zygimantus"]
```
Comment (vendethiel) [inline lt-lt/tmux-lt.html.markdown]: please restore the contributors tag and add yourself as a translator instead

Comment (zygimantus) [inline lt-lt/tmux-lt.html.markdown]: What do you mean?

Comment (vendethiel) [inline lt-lt/tmux-lt.html.markdown]: This is a translation of the english tmux article right?

Comment (zygimantus) [inline lt-lt/tmux-lt.html.markdown]: Yes. I understood now.

**Comment by zygimantus** [comment]
I fixex this.

**Comment by vendethiel** [comment]
Thanks

**Pull request merged**